src/Features/Core/Portable/UnifiedSuggestions/UnifiedSuggestedActionsSource.cs

```diff
@@ -284,7 +284,7 @@ static CodeFixGroupKey GetGroupKey(CodeFix fix)
             {
                 // Wrap the suppression/configuration actions within another top level suggested action
                 // to avoid clutter in the light bulb menu.
-                var suppressOrConfigureCodeAction = new NoChangeAction(CodeFixesResources.Suppress_or_Configure_issues);
+                var suppressOrConfigureCodeAction = new NoChangeAction(CodeFixesResources.Suppress_or_Configure_issues, CodeActionPriority.Low);
```

Comment:
Explicitly changed the priority of "Suppress or Configure issues" to low so that it will always show at the bottom of the code actions list, matching the local experience.

Reply:
shouldn't this be copying an existing priority over?  i.e. instead of hardcoding this in here, we shoudl grab the priority of the actual code action that the other parts of our system is generating?